src/shared/bootspec.c

```diff
@@ -505,6 +505,12 @@ static int boot_entry_compare(const BootEntry *a, const BootEntry *b) {
         assert(a);
         assert(b);
 
+        /* This mimics a funciton of the same name in src/boot/efi/sd-boot.c */
```

Comment:
Typo

Reply:
you have to admit that "funkyton" sounds much nicer though!